backend/game/player/player.py

```diff
@@ -37,6 +37,12 @@ def update_energy_by(self, change_integer):
         if self.energy < constants.DEFAULT_ENERGY_CUBE:
             self.energy = constants.DEFAULT_ENERGY_CUBE
 
+    def lose_all_stars(self):
+        self.victory_points = constants.DEATH_HIT_POINT
+
+    def discard_all_cards(self):
+        self.cards = []
```

Comment:
Cards shouldn't just disappear, they need to go back to the discard pile

Reply:
I agree though not sure where this discard pile exists, I'll make a change to use the remove method and list comprehension but then I think all cards that need to be removed should use a discard method on the card

Reply:
There's a `deckhandler.discard()` method, but not exactly sure how to attach that yet